Exiled.API/Enums/EffectType.cs

```diff
@@ -199,5 +199,25 @@ public enum EffectType
         /// Teleports the player to the pocket dimension and drains health until the player escapes or is killed.
         /// </summary>
         PocketCorroding,
+
+        /// <summary>
+        /// Reduces walking sound by 10%.
+        /// </summary>
+        SilentWalk,
+
+        /// <summary>
+        /// Makes you a marshmallow guy.
+        /// </summary>
+        Marshmallow,
+
+        /// <summary>
+        /// The effect that receive player when he getting attacked by SCP-3114 ability.
+        /// </summary>
+        Strangled,
+
+        /// <summary>
+        /// Allows you to bypass doors.
```

Comment:
```suggestion
        /// Makes the player nearly invisible, and allows them to pass through doors.
```
Documentation nitpick